db/seeds.rb

```diff
@@ -5,3 +5,107 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# Users
+users = [
+    {
+        email: 'guerrero@email.com',
+        password: 'password'
+    },
+    {
+        email: 'fernando@email.com',
+        password: 'password'
+    },
+    {
+        email: 'lex@email.com',
+        password: 'password'
+    },
+    {
+        email: 'allen@email.com',
+        password: 'password'
+    }
+]
+
+users.each do |attributes|
+    User.create attributes
+end
+
+
+# Cards
+response = RestClient.get 'https://api.pokemontcg.io/v2/cards?q=set.id:base1'
+json = JSON.parse response
+puts json
+if !json.nil?
+  json["data"].map do |card|
+    if card["supertype"] == "Pokémon"
+      pokemon_type = card["types"][0]
+    else
+      pokemon_type = card["supertype"]
+    end
+    if card.key?("tcgplayer") == false
```

Comment:
```suggestion
    if card.key?("tcgplayer")
```
`.key?()` is already returning a boolean so `== false` is redundant